tasks/main.yml

```diff
@@ -18,6 +18,7 @@
     group: root
     mode: 0644
   notify: restart nfs
+  when: nfs_disable_server == false
```

Comment:
I don't follow. Why that condition needs to be added to the next task? if nfs_exports is not empty we need to run nfs server. If it's empty why do we need to run nfs-server?